packages/ses/src/module-load.js

```diff
@@ -145,83 +139,72 @@ function* loadWithoutErrorAnnotation(
   const { importHook, importNowHook, moduleMap, moduleMapHook, moduleRecords } =
     weakmapGet(compartmentPrivateFields, compartment);
 
+  if (mapHas(moduleRecords, moduleSpecifier)) {
```

Comment:
The meat of this change begins here thru the end of `loadWithErrorAnnotation`, which preserves all prior behaviors, fixes a bug or two, and prepares to add new module descriptor shapes in order to be more compatible with usage of XS.

This is preparation to align SES with XS module descriptors https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/xs/XS%20Compartment.md#-module-descriptor

Reply:
I don't see a `loadWithErrorAnnotation`.

Did you mean `loadWithoutErrorAnnotation` or `memoizedLoadWithErrorAnnotation`?

Reply:
Pardon, the surrounding function, `loadWithoutErrorAnnotation`.

Reply:
Resolving